frontend/src/views/MigrationHistoryDetail.vue

```diff
@@ -199,21 +235,42 @@ export default defineComponent({
   setup(props) {
     const store = useStore();
 
-    const migrationHistory = computed((): MigrationHistory => {
-      return store.getters["instance/migrationHistoryById"](
-        idFromSlug(props.migrationHistorySlug)
+    const database = computed((): Database => {
+      return store.getters["database/databaseById"](
+        idFromSlug(props.databaseSlug)
+      );
+    });
+
+    // get all migration histories before (include) the one of given id, ordered by descending version.
+    const prevMigrationHistoryList = computed((): MigrationHistory[] => {
```

Comment:
If I understand correctly, we may only need to query the current selected migration history and the one right before it?

Reply:
yes, but we use `store.getters` here. it gets results from the early cache and does not cause network overhead. I think it's okay.